docs/source/notebooks/api_quickstart.ipynb

```diff
@@ -16,12 +16,13 @@
      "name": "stdout",
```

Comment:
<ul><li>After this last cell, can you add a last cell using <a href="https://github.com/rasbt/watermark" target="_blank">the watermark package</a>:</li></ul><p> </p><pre class="ql-syntax" spellcheck="false">%load_ext watermark
%watermark -n -u -v -iv -w
</pre><ul><li>Finally, can you <a href="https://github.com/dfm/black_nbconvert" target="_blank">format the NB with Black</a>: <code>pip install black_nbconvert</code> and then <code>black_nbconvert /path/to/a/notebook.ipynb</code> on the command line -- this will reformat the NB inplace automatically 😉</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='105'/>